test/autest/gold_tests/autest-site/txn_box.test.ext

```diff
@@ -104,6 +63,7 @@ def TxnBoxTestRun(self, text, replay_path, **kw):
         # The following is needed for ATS 9 and later.
         # 'proxy.config.plugin.dynamic_reload': 0
     })
+    ts.Disk.remap_config.AddLine('map / http://127.0.0.1:{0}'.format(pv_server.Variables.http_port))
```

Comment:
Must be able to inhibit this, otherwise adding other remap rules will useless, as all transactions will match this one first.

Reply:
This is addressed by your TxnBoxTestRun remap parameter suggestion.